scopes/harmony/cli/command-runner.ts

```diff
@@ -134,3 +131,12 @@ export class CommandRunner {
     return null;
   }
 }
+
+function toRenderResult(obj: RenderResult | React.ReactElement) {
+  return isRenderResult(obj) ? obj : { data: obj, code: 0 };
+}
+
+function isRenderResult(obj: RenderResult | any): obj is RenderResult {
+  // eslint-disable-next-line no-prototype-builtins
+  return typeof obj === 'object' && typeof obj.code === 'number' && !!obj.hasOwnProperty('data');
```

Comment:
@KutnerUri , why `!!` before `obj.hasOwnProperty` ? it returns boolean.

Reply:
huh. I'll fix.
```suggestion
  return typeof obj === 'object' && typeof obj.code === 'number' && obj.hasOwnProperty('data');
```